### PR TITLE
Update HelmDeployV0 kubeconfig file for multiple tasks scenarios

### PR DESCRIPTION
**Pull request by qianz2**
**Task name**: HelmDeployV0 

**Description**:  kubeConfig changes related to this issue (https://github.com/microsoft/azure-pipelines-tasks/issues/18666)
Support mulitple HelmDeployV0 tasks with external authentication scenarios. Stop cleaning out kubeconfig files once a task is ended, so that later HelmDeployV0 tasks can be run using the same kube config file.

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "shelljs": "^0.8.5",
     "sync-request": "4.1.0",
     "typed-rest-client": "^1.8.9",
-    "typescript": "4.0.2",
+    "typescript": "^4.0.2",
```
Comment (merlynomsft) [inline package.json:48]: change needed?

Comment (qianz2) [reply package.json:48]: It's auto generated after running `Node make.js build --task HelmDeployV0` command.

Comment (qianz2) [reply package.json:48]: I checked no errors if I removed this change, so it may be safe to remove it.

**Review by merlynomsft — APPROVED** [review]

**Pull request merged**